README.md

```diff
@@ -60,7 +53,7 @@ class Counter extends HTMLElement {
 window.customElements.define('num-counter', Counter);
 ```
 
-In the above example, you can see a field declared with the syntax `x = 0`. You can also declare a field without an initializer as `x`. By declaring fields up-front, class definitions become more self-documenting; instances go through fewer state transitions, as declared fields are always present.
+In the above example, you can see two class fields declared: `x = 0;` and `onclick = () => { /* ... * /};`. You can also declare a field without an initializer as `x`. By declaring fields up-front, class definitions become more self-documenting; instances go through fewer state transitions, as declared fields are always present.
```

Comment:
this note is no longer true with your change - the requirement is that the declared field is present *on the prototype* as well as being bound on the instance.

Reply:
> this note is no longer true with your change

there are quite a few statements on the diffed line. Which one in particular is no longer true?

Reply:
“As declared fields are always present”

Reply:
I don't understand. How can something be true for
```
class { x = 0; }
```
but not true for
```
class { onclick = () => {}; }
```
?